src/main/java/ui/UI.java

```diff
@@ -143,6 +143,14 @@ private void login(boolean isBypassLogin) {
         getMainWindowHandle(mainStage.getTitle());
     }
 
+    private void launchSampleBoard(){
+        BoardAutoCreator boardCreator = new BoardAutoCreator(this, panels, prefs);
+        if (prefs.getAllBoardNames().contains(boardCreator.SAMPLE_BOARD)){
+            prefs.removeBoard(boardCreator.SAMPLE_BOARD);
```

Comment:
I was also thinking of possibility of duplicates, which is also not possible since this is a first time user we are talking about!